src/app/conf/2024/_videos.ts

```diff
@@ -0,0 +1,280 @@
+export const videos = [
+  {
+    id: "b45lWgkVLYA",
```

Comment:
We need to link to the new channel's videos - @jorydotcom and I will action this :+1:

Reply:
Done; thanks @jemgillam